dnf.spec

```diff
@@ -189,6 +189,16 @@ Requires:       python3-%{name} = %{version}-%{release}
 %description automatic
 Systemd units that can periodically download package upgrades and apply them.
 
+%package bootc
```

Comment:
Maybe the new dnf-bootc package should require "python3-%{name} = %{version}-%{release}". Otherwise people could install dnf-bootc without python3-dnf. Quite pointless. Also one could install different versions of python3-dnf not matching the new (ostree etc.) dependencies.

Reply:
Good idea! Done.